core/main.go

```diff
@@ -61,22 +74,42 @@ func deployCoreServices(c *config.Config, api *api.API) error {
 	return nil
 }
 
+func stopRunningServices(api *api.API) error {
+	services, err := api.ListServices()
+	if err != nil {
+		return err
+	}
+	for _, s := range services {
```

Comment:
Could all the services be stop at the same time in parallel?

Reply:
It could I just wanted to make it simple for now. This can be done in another PR